README.md

```diff
@@ -30,8 +30,7 @@ The user-friendly interface allows for straightforward secure setup and manageme
 
 ## 🏷️ Supported providers
 
-`Strato` `DDNSS` `Dynu` `Aliyun` `NoIP` `DD24` ~~`AllInkl`~~ ~~`Cloudflare`~~ ~~`DigitalOcean`~~ ~~`DonDominio`~~ ~~`DNSOMatic`~~ ~~`DNSPod`~~ ~~`Dreamhost`~~ ~~`DuckDNS`~~ ~~`DynDNS`~~ ~~`FreeDNS`~~ ~~`Gandi`~~ ~~`GCP`~~ ~~`GoDaddy`~~
-~~`Google`~~ ~~`He.net`~~ ~~`Infomaniak`~~ ~~`INWX`~~ ~~`Linode`~~ ~~`LuaDNS`~~ ~~`Namecheap`~~  ~~`Njalla`~~ ~~`OpenDNS`~~ ~~`OVH`~~ ~~`Porkbun`~~ ~~`Selfhost.de`~~ ~~`Servercow.de`~~ ~~`Spdyn`~~ ~~`Variomedia.de`~~
+`Strato` `DDNSS` `Dynu` `Aliyun` `NoIP` `DD24` `INWX`
 
 > **Note:** The crossed out providers will be implemented in future releases. In addition, the implementation of an individual update link with user-specific input and return values is planned.
 
```

Comment:
You might wanna consider removing this note if no crossed out providers exist. Just a suggestion, tho

Reply:
Fixed in [f3f0c7d](https://github.com/Plaenkler/DDNS-Updater/pull/153/commits/f3f0c7d14beddd903b7550cbaf5bb661ed5e55d9)